<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C-- IDE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tw_main"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="new_action"/>
    <addaction name="save_action"/>
    <addaction name="open_action"/>
   </widget>
   <widget class="QMenu" name="menu_run">
    <property name="title">
     <string>运行</string>
    </property>
    <addaction name="run_action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_run"/>
  </widget>
  <action name="new_action">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="open_action">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="run_action">
   <property name="text">
    <string>运行</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
